standup/api/views.py

```diff
@@ -70,6 +73,12 @@ def __init__(self, content, status=None, cors=False):
             self['Access-Control-Allow-Origin'] = '*'
 
 
+@require_safe
```

Comment:
I never knew that existed. I wonder if we should use this for other API endpoints.

Reply:
I think the others allow POST, so they need to allow most HTTP verbs. This is a read-only endpoint however. I like `require_safe` over `require_GET` since it also specifically allows `HEAD` requests.